README.md

```diff
@@ -1,2 +1,70 @@
-# enhancements
-Enhancement Tracking and Backlog Repo for Istio Releases
+# Enhancement Tracking and Backlog
+Enhancement Tracking and Backlog Repo for Istio Releases. Owned by TOC.
+
+This repo contains issues and Istio enhancement proposal. These issues are umbrellas for new enhancements to be added to Istio. An enhancement usually takes multiple releases to complete. And an enhancement can be tracked as backlog items before work begins. An enhancement may be filed once there is consensus in at least one Istio working group.
+
+## Why do we need this?
+
+As a project it is vital to be able to track the chain of custody for a proposed enhancement from conception through implementation. The enhancement repository provides a standardized mechanism for describing important enhancements delivered through Istio releases.  This will not only enable the contributors, maintainers and release managers to explaining features to users coherently but also allow adopters to plan their adoption strategy based on a forward looking road map.
+
+## Is My Thing an Enhancement?
+
+We are trying to figure out the exact shape of an enhancement. Until then here are a few rough heuristics.
+
+An enhancement is anything that:
+
+- a blog post would be written about after its release (ex. [v1beta1 authorization policy](https://istio.io/blog/2019/v1beta1-authorization-policy/))
+- requires multiple parties/SIGs/owners participating to complete (ex. auto mTLS)
+- will be graduating from one stage to another (ex. alpha to beta, beta to GA)
+- needs significant effort or changes Istio in a significant way (ex. something that would take 10 person-weeks to implement, introduce or redesign a system component, or introduces API changes)
+- impacts the UX or operation of Istio substantially such that engineers using Istio will need retraining
+- users will notice and come to rely on
+
+It is unlikely an enhancement if it is:
+- fixing a flaky test
+- refactoring code
+- performance improvements, which are only visible to users as faster API operations, or faster control loops
+- adding error messages or events
+
+If you are not sure, ask someone in the working group where you initially circulated the idea.
+
+## When to Create a New Enhancement Issue
+
+Create an issue here once you:
+- have circulated your idea to see if there is interest
+   - through Community Meetings, working group meetings, discuss.istio.io lists, or an issue in github.com/istio/istio
+- (optionally) have done a prototype in your own fork
+- (optionally) have identified people who agree to work on the enhancement
+  - many enhancements will take several releases to progress through Alpha, Beta, and Stable stages
+  - you and your team should be prepared to work on the approx. 9mo - 1 year that it takes to progress to Stable status
+- (optionally) are ready to be the project-manager for the enhancement
```

Comment:
Does a KEP always imply the creators of the KEP will work on it? Or is it valid to create a full design doc + roadmap but not commit to work on it?

Reply:
that is totally valid.  I think KEP does require it has engineers allocated to it... We don't necessarily want that... of course, without resource, it will not be placed in any release.